e2e/functionalities/peer-dependency-component.e2e.ts

```diff
@@ -0,0 +1,49 @@
+import path from 'path';
+import { expect } from 'chai';
+import fs from 'fs-extra';
+import Helper from '../../src/e2e-helper/e2e-helper';
+
+describe('set-peer', function () {
+  this.timeout(0);
+  let helper: Helper;
+  before(() => {
+    helper = new Helper();
+  });
+  after(() => {
+    helper.scopeHelper.destroy();
+  });
+  describe('a component is a peer dependency', () => {
+    let workspaceCapsulesRootDir: string;
+    before(() => {
+      helper.scopeHelper.reInitLocalScope();
+      helper.fixtures.populateComponents(2);
+      helper.command.setPeer('comp2', '0');
+      helper.command.install();
+      helper.command.build();
+      workspaceCapsulesRootDir = helper.command.capsuleListParsed().workspaceCapsulesRootDir;
+    });
+    it('should save the peer dependency in the model', () => {
```

Comment:
I would add another test with snap/tag and validate the component data after it
for both the always peer component (make sure that data of always peer is stored) and the dependent component (make sure it was stored with peer dep)

Reply:
done